packages/frontend/src/components/CFTarget.vue

```diff
@@ -103,6 +108,22 @@ export default {
       }
     },
   },
+  mounted() {
+    console.log("orgMissing changed:");
+    this.$nextTick(() => {
+      const orgSelect = this.$refs.orgSelect;
+      if (orgSelect && orgSelect.shadowRoot && !orgSelect.shadowRoot.querySelector("style[data-org-option-style]")) {
+        const style = document.createElement("style");
+        style.setAttribute("data-org-option-style", "true");
+        style.textContent = `
+              li.option {
+                height: auto;
+              }
+            `;
+        orgSelect.shadowRoot.appendChild(style);
+      }
+    });
+  },
   updated() {
```

Comment:
consider setting up the `space` dropdown in the same way - it might happen there too